src/configuration.rs

```diff
@@ -48,9 +52,9 @@ impl Configuration {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Base {
-    pub bind_address: String,
-    pub port: i32,
-    pub port_ssl: i32,
+    pub bind_address: IpAddr,
```

Comment:
You changed the type of bind_address, is it not a problem for the YAML configuration file?

Reply:
Serde implements Deserialize for IpAddr, and it seems like it just uses the appropriate from_str function internally (or at least it parses the previous configuration correctly).

Reply:
Okey great